01-Navigation/4-Contextual_navigation/4.2-Navigate_To_menu_on_type.cs

```diff
@@ -4,38 +4,64 @@
+    // How to use Navigate to
+    // <shortcut id="Navigate to">Alt+`        (ReSharper VisualStudio Keymap)</shortcut>
+    // <shortcut id="Navigate to">Ctrl+Shift+G (Rider Default IntelliJ Keymap)</shortcut>
     //
-    // <shortcut id="Navigate To...">Alt+`</shortcut>
+    // Put the caret on the target and let's run it.
     //
 
-
     // 1. When the caret is on a type definition (DerivedClass)
-    //    (Find Usages, related files covered later)
-    //    a) Go to Base Symbols + Derived Symbols
-    //    b) Extension methods
-    //    c) Object Browser, Windows Explorer, Command Prompt
-    //    d) Locate in Solution Explorer
-    //    d) Referenced Code finds all code referenced by the type, e.g. BaseClass
+    //    The following candidates will be displayed.
+    //      - Declaration           : Navigate to Declaration
+    //      - Implementation        : Navigate to Implementation
+    //      - Base Symbols          : Navigate to Base Class
+    //      - Find Usages of Symbol : (*1)
+    //      - Related Files...      : (*1)
+    //      - Usages of Symbol      : (*1)
+    //      - Derived Symbols       : Navigate to Derived Class
+    //      - Extension methods     : Navigate to Extension methods
+    //      - Consuming APIs        : Navigate to the method that is taking itself as a parameter
+    //      - Exposing APIs         : Navigate to the method that returns itself as the return value
+    //      - Referenced Code       : Find all the code where the type is referenced ( e.g. BaseClass )
+    //      - Locate in Solution Explorer   : (Only ReSharper) Navigate to current file in Solution Explorer
+    //
+    //    (*1) Find Usages of Symbol, Usages of Symbol, and Related files
+    //         will be explained in detail in later workshop.
+    //
     public class DerivedClass   // <- Text caret on DerivedClass
         : BaseClass
     {
     }
 
 
     // 2. When the caret is on a base class (BaseClass)
-    //    a) As above (find usages, base symbols, etc.)
-    //    b) Declaration navigates to the declaration of BaseClass
-    //    c) Implementation navigates to the implementations of BaseClass - deriving symbols
-    //    d) Overriding members highlights all members that override from BaseClass
-    //       Escape clears the highlight
+    //    The candidates will be displayed again.
+    //
+    //    In the case of this class,
+    //    since it has overridden method definitions,
+    //    ReSharper and Rider will correctly recognize it
+    //    and display it as a candidate.
+    //      - Overriding members    : Highlight all overridden methods
+    //                                Escape clears the highlight
+    //
+    // (Reference) How to move between candidates
+    //
+    // NOTE: Rider shortcuts are undefined by default (no configuration) and must be configured to work.
+    // [Settings] > [Keymap] > [Main Manu] > [Edit] > [Find Usages]
```

Comment:
Quick style update (we use `|` to separate menus in all of our docs so might make sense to keep this in the workshop, too)

```suggestion
    // Settings | Keymap | Main Menu | Edit | Find Usages
```